behat/tests/bootstrap/v2/reporting/sections/ReportingSectionsFeatureContext.php

```diff
@@ -65,8 +65,7 @@ public function nextSectionShouldBe(string $sectionName)
      */
     public function iNavigateBackToReportSection()
     {
-        $linkText = $this->reportUrlPrefix == 'ndr' ? 'New deputy report overview' : 'Deputy report overview';
-        $this->clickLink($linkText);
+        $this->iClickBasedOnElementId('a', 'data-action', 'report.overview');
```

Comment:
This could make our lives a lot easier in many places where we have to narrow down which link we want - worth keeping in mind for future tickets too. Feels nicer than the classes adding to links in old-style tests.